src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/SqlJournal.cs

```diff
@@ -106,7 +106,7 @@ protected override bool ReceivePluginInternal(object message)
                     return true;
                 case SelectCurrentPersistenceIds request:
                     SelectAllPersistenceIdsAsync(request.Offset)
-                        .PipeTo(request.ReplyTo, success: result => new CurrentPersistenceIds(result.Ids, request.Offset));
```

Comment:
This is a major bug, if we kept sending `request.Offset`, then we will always retrieve ALL of persistence ids from offset 0 on all `SelectCurrentPersistenceIds` command. `result.LastOrdering` is the proper `HighestOrderingNumber`.

Reply:
Nice catch